pkg/config/store/store2_test.go

```diff
@@ -100,12 +109,19 @@ func TestStore2(t *testing.T) {
 		t.Error(err)
 	}
 	m.ch <- BackendEvent{
-		Key:   k,
-		Type:  Update,
-		Value: map[string]interface{}{"name": "default", "adapter": "noop"},
-	}
-	wantEv := Event{Key: k, Type: Update, Value: want}
-	if ev := <-ch; !reflect.DeepEqual(ev, wantEv) {
+		Key:  k,
+		Type: Update,
+		Value: &BackEndResource{
+			Spec: map[string]interface{}{"name": "default", "adapter": "noop"},
+		},
+	}
+	wantEv := Event{Key: k, Type: Update,
+		Value: &Resource{
+			Spec: want,
+		},
+	}
+	ev := <-ch
+	if !reflect.DeepEqual(ev, wantEv) {
```

Comment:
I prefer the previous structure -- `if ev := <-ch; !reflect.`... would be fine (and more concise), I think)